apps/framework-cli/src/cli/display/infrastructure.rs

```diff
@@ -15,6 +16,113 @@ use crate::framework::core::{
 use crossterm::{execute, style::Print};
 use log::info;
 
+/// Create the detail indentation string at compile time
+/// Computed from ACTION_WIDTH (15) + 3 spaces:
+/// - ACTION_WIDTH spaces for the action column
+/// - 1 space after the action symbol (e.g., "+", "-", "~")  
+/// - 2 spaces for additional indentation of detail lines
+///   Total: 18 spaces for proper alignment
+const DETAIL_INDENT: &str = {
+    const INDENT_BYTES: [u8; ACTION_WIDTH + 3] = [b' '; ACTION_WIDTH + 3];
+    // This is safe because we know all bytes are spaces (0x20), which are valid UTF-8
+    match std::str::from_utf8(&INDENT_BYTES) {
+        Ok(s) => s,
+        Err(_) => panic!("Invalid UTF-8 in DETAIL_INDENT"),
```

Comment:
maybe just use `.expect` here?

Reply:
Just tried and that doesn't work in this context

```
cannot call non-const method `std::result::Result::<&str, std::str::Utf8Error>::expect` in constants
calls in constants are limited to constant functions, tuple structs and tuple variants
```